README.md

```diff
@@ -166,6 +166,7 @@ brew install python3 cmake pkg-config gtksourceviewmm3 gnome-icon-theme gspell l
 Get cherrytree source, compile and run:
 ```sh
 git clone https://github.com/giuspen/cherrytree.git
+cd cherrytree
 git submodule update --init
 mkdir cherrytree/build
```

Comment:
I caused the issue adding the git command, but with your cd cherrytree you invalidated the following two, has to become mkdir build && cd build, could you fix that too? May I ask you to do the same for the other places in the file where I added git submodule update --init?

Reply:
Sure I'll add the other ones too